src/aks-preview/azext_aks_preview/custom.py

```diff
@@ -608,8 +618,38 @@ def aks_create(
     except DecoratorEarlyExitException:
         # exit gracefully
         return None
-    # send request to create a real managed cluster
-    return aks_create_decorator.create_mc(mc)
+
+    try:
```

Comment:
should use the create method from the decorator

Reply:
updated, thanks for the feedback!